### PR TITLE
Optional ideas

### PR DESCRIPTION
**Pull request by WhatisRT**
This moves a bunch of stuff in the optional section

### CODE REVIEW

File: CIP-1694/README.md [hunk 1]
```diff
@@ -228,6 +228,31 @@ How any private companies, public or private institutions,  individuals etc. cho
   - [Changes to the stake snapshot](#changes-to-the-stake-snapshot)
   - [Definitions relating to voting stake](#definitions-relating-to-voting-stake)
 
+### Definitions
+
+Eligible stake: all stake that belongs to a registered stake credential.
+DRep: registered DRep or auto DRep
+Active DRep stake: Eligible stake that is delegated to a DRep
+Active reward stake: Eligible stake delegated to an active stake pool
+
```
Comment (polinavino) [inline CIP-1694/README.md:237]: This whole section below is about counting votes, it should have a separate title, eg. 
### Counting votes

File: CIP-1694/README.md [hunk 2]
```diff
@@ -228,6 +228,31 @@ How any private companies, public or private institutions,  individuals etc. cho
   - [Changes to the stake snapshot](#changes-to-the-stake-snapshot)
   - [Definitions relating to voting stake](#definitions-relating-to-voting-stake)
 
+### Definitions
+
+Eligible stake: all stake that belongs to a registered stake credential.
+DRep: registered DRep or auto DRep
```
Comment (polinavino) [inline CIP-1694/README.md:234]: auto DRep is confusing by itself. Maybe call it pre-set DRep option, or just link to section describing what it is

File: CIP-1694/README.md [hunk 3]
```diff
@@ -1129,6 +1127,28 @@ We could reduce this deposit for proposals which go through some agreed upon off
...
 
+##### Different deposit amounts for different governance actions
+
+Multiple workshops for this CIP have proposed to introduce a different
+deposit amount for each type of governance action. It was not clear
+whether a majority was in favor of this idea, but this may be
+considered if it becomes clear that it is necessary.
+
+##### Minimum AVS
+
```
Comment (polinavino) [inline CIP-1694/README.md:1138]: what does AVS stand for?

File: CIP-1694/README.md (hunk 1 above)
Comment (WhatisRT) [reply CIP-1694/README.md]: This commit shouldn't have been in this PR, it's already on your branch

File: CIP-1694/README.md (hunk 2 above)
Comment (WhatisRT) [reply CIP-1694/README.md]: Same as above

File: CIP-1694/README.md (hunk 3 above)
Comment (WhatisRT) [reply CIP-1694/README.md:1113]: I modified the header, it stands for 'active voting stake'

**Pull request merged**